.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: go
+dist: trusty
```

Comment:
It passed for me after adding sudo line  https://travis-ci.org/allegro/marathon-consul/builds/195645241